libtransmission/utils.mm

```diff
@@ -0,0 +1,37 @@
+// This file Copyright © 2022-2023 Mnemosyne LLC.
+// It may be used under GPLv2 (SPDX: GPL-2.0-only), GPLv3 (SPDX: GPL-3.0-only),
+// or any future license endorsed by Mnemosyne LLC.
+// License text can be found in the licenses/ folder.
+
+#import <Foundation/Foundation.h>
+#include <string>
+#include <string_view>
+#include <utf8.h>
```

Comment:
_(nit)_: include order should be sorted by level of canon, with a paragraph break between. In this case, language standard -> platform API -> third-party cpputf

```suggestion
#include <string>
#include <string_view>

#import <Foundation/Foundation.h>

#include <utf8.h>
```

Reply:
It wasn't in this order in tr-assert.mm, and it's classic to place Foundation on top (you could say it's part of the ObjC++ language). But I'll add the paragraph breaks, sure. 👍